reference/shipping.v2.yml

```diff
@@ -733,6 +733,7 @@ paths:
                       handling_fees:
                         fixed_surcharge: "0"
                       is_fallback: false
+                      channels: [2,3]
```

Comment:
[2, 3]
here and below
or is the lack of spacing a standard here?

Reply:
Hmm thanks Ben, I was mistaken it would be different with the docs, and though in other areas in the docs there are ones with no spac, I think we can apply styling best practice here as pioneers😆🍻